docs/reference.rst

```diff
@@ -57,6 +57,23 @@ TensorNodes
         :special-members:
         :exclude-members: __init__, __weakref__
 
+Converter
+^^^^^^^^^
+
+`nengo_dl.Converter` can be used to automatically convert a Keras model to a native
```

Comment:
The problem is that you can't have two ``automodule`` directives for the same module, or sphinx complains. And we're already using this in the "Layer converters" section in the backend docs down below (I used it there since there are more objects being autodocumented).

Reply:
Ah interesting, that makes sense. Disregard!